builders/get-started/eth-compare/security.md

```diff
@@ -15,11 +15,13 @@ This guide will outline and provide examples of some security considerations to
 
 ## Arbitrary Code Execution {: #arbitrary-code-execution }
 
-Smart contracts can allow arbitrary code execution through low-level message passing to other smart contracts via the [`call()` function](https://solidity-by-example.org/call/){target=_blank}, which is an available method for the [address data type in Solidity](https://docs.soliditylang.org/en/latest/types.html#address){target=_blank}.
+Arbitrary code execution in Solidity is the ability to execute code and call functions of other contracts using an arbitrary number of arguments of any type.
```

Comment:
How about something like "A smart contract allows arbitrary execution of other contract code when it allows user-specified data when invoking a `call()`.

That probably needs a lot of work, but I think it includes some elements that you're trying to convey.

Specifically, I think we want to make it clear that we're talking about a smart contract making a call (as opposed to call from a EOA via a txn) in ways that are unintended.

Reply:
Here's the pattern we really care about (in its simplest form):

```
EOA signs transaction -> calls contract A -> calls contract B
```

This is problematic any time `contract A` allows user-input to influence its call to `contract B` -- this is true in Ethereum proper, of course, but we define the additional caveats that are introduced by Moonbeam here.

^ really just me summarizing the problem. What I think is missing in the discussion of "arbitrary code execution" is clarity on the pattern (specifically A -> B with some user-influence).